react/ToggleContent/ToggleContent.less

```diff
@@ -0,0 +1,5 @@
+@import (reference) '~seek-style-guide/theme';
+
+.button {
+  box-shadow: none !important;
```

Comment:
Looks like box shadow has already been set to none. Do we need to declare it again?

![screen shot 2018-12-17 at 9 47 19 am](https://user-images.githubusercontent.com/1896277/50059892-d63af180-01e0-11e9-8709-5734bc058365.png)

Reply:
we need to overwrite focus state coming from the button, I'll just do that directly

Reply:
I'm confused - why would we want to override that style? If we turn off the box shadow there's no way to tell your focussed on the button when using the page with a keyboard

Reply:
But that breaks keyboard accessibility of the button. Why do you want to reset it?